src/pages/Login.js

```diff
@@ -0,0 +1,93 @@
+import Post from "../components/UI/Post";
+import classes from "./Login.module.css";
+import action from "../components/posts/buttons/Button.module.css";
+import { useNavigate } from "react-router-dom";
+import { useRef, useState } from "react";
+import CreateLoginForm from "../components/login/CreateLogin";
+import Backdrop from "../components/UI/Backdrop";
+import { API_KEY } from "../Local-config";
+
+function LoginPage({ setShowError, setErrorText, setIdToken }) {
+  const [showCreateLogin, setShowCreateLogin] = useState(false);
+  const [emailError, setEmailError] = useState("");
+  const navigate = useNavigate();
+  const emailRef = useRef();
+  const passRef = useRef();
+  let fireResp = false;
+
+  function onSubmit(event) {
+    event.preventDefault();
+
+    const account = {
+      email: emailRef.current.value,
+      password: passRef.current.value,
+      returnSecureToken: true,
+    };
+
+    fetch(
+      `https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword?key=${API_KEY}`,
+      {
+        method: "POST",
+        body: JSON.stringify(account),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    )
+      .then((response) => {
+        if (response.status === 200) {
+          fireResp = true;
+        } else {
+          fireResp = false;
+        }
+        return response.json();
+      })
+      .then((data) => {
+        if (fireResp === true) {
+          setIdToken(data.idToken);
+          navigate("/feed");
+        } else {
+          setEmailError(data.error.message);
+        }
+      });
+  }
+
+  function createLoginHandler() {
+    setShowCreateLogin(!showCreateLogin);
+  }
+  return (
+    <>
+      <form className={classes.login} onSubmit={onSubmit}>
+        <Post>
+          <div className={classes.content}>
+            <div className={classes.input}>
+              <input required placeholder="Email" ref={emailRef}></input>
+              <input required placeholder="Password" ref={passRef}></input>
```

Comment:
Might be nice if the password is hidden behind `*`s or dots when displayed, might be manageable using hooks and states or a configure option